server/src/schemas/storeFrontSchema.ts

```diff
@@ -0,0 +1,74 @@
+import { gql } from "apollo-server-express";
+
+export const StoreFrontTypeDefs = gql`
+
+	scalar Date
+
+	type InventoryItem {
+		id: ID!;
+		image: String;
+		name: String;
+		labels: [String];
+		unit: String;
+		pluralUnit: String;
+		count: Int;
+		pricePerUnit: Float;
+	}
+
+	input InventoryItemInput {
+		image: String;
+		name: String;
+		labels: [String];
+		unit: String;
+		pluralUnit: String;
+		count: Int;
+		pricePerUnit: Float;
+	}
+
+	type PurchaseOrderItem {
+		id: ID!;
+		item: InventoryItem;
+		count: Int;
+	}
+
+	type PurchaseOrder {
+		id: ID!;
+		creator: Person;
+		createDate: Date;
+		expectedDeliveryDate: Date;
+		items: [PurchaseOrderItem];
+		attachments: [String];
+	}
+
+	input PurchaseOrderInput {
+		creator: Person;
+		createDate: Date;
+		expectedDeliveryDate: Date;
+		items: [PurchaseOrderItem];
+		attachments: [String];
+	}
+
+	# might belong in a different file but just gonna throw it here for now
+	type Person {
+		id: ID!;
+		name: String;
+		avatarHref: String;
```

Comment:
For consistency throughout the schema, I think we should rename `avatarHref` to `image`.